supermarq-benchmarks/examples/qcvv/qcvv_irb_css.ipynb

```diff
@@ -0,0 +1,164 @@
+{
```

Comment:
<p>We could also link the IRB paper: <a href="https://arxiv.org/abs/1203.4550" target="_blank">https://arxiv.org/abs/1203.4550</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/994/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I've added this to the docstring of the class rather than the notebook as that feels like a more natural place for it. I've also added a reference for the XEB routine.</p>